libvmi/libvmi.h

```diff
@@ -1856,8 +1900,23 @@ unsigned int vmi_get_num_vcpus (
  */
 status_t vmi_get_vcpureg(
     vmi_instance_t vmi,
-    reg_t *value,
-    registers_t reg,
+    uint64_t *value,
+    reg_t reg,
+    unsigned long vcpu);
+
+/**
+ * Gets the current value of a VCPU registers.  This currently only
+ * supports x86 registers.  When LibVMI is accessing a raw
+ * memory file or KVM, this function will fail.
```

Comment:
Typo fixed. For ARM we would need an arm_regs struct defined for registers similar to the x86 regs we use here.